src/main/resources/application-dev.yml

```diff
@@ -1,6 +1,6 @@
 waypoint:
   dashboard:
-    base-url: http://host.docker.internal:8083
```

Comment:
Let's revert the content of this file also. You can ignore pushing up this file in the future as it contains the configs specific to your system